src/YQMenuBar.cc

```diff
@@ -80,6 +80,8 @@ YQMenuBar::rebuildMenuTree()
 
 	QMenu * menu = QMenuBar::addMenu( fromUTF8( item->label() ));
 	item->setUiItem( menu );
+        menu->setEnabled( item->isEnabled() );
+        menu->menuAction()->setVisible( item->isVisible() );
```

Comment:
NP, maybe github issue but indentation looks inconsistent. ( maybe mix of tabs and spaces? )

Reply:
GitHub could never handle tabs properly.